src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,47 @@
+import { Todo } from '../../types/Todo';
+import { PageNavLink } from '../PageNavLink';
+
+type Props = {
+  todos: Todo[],
+  clearCompleted: () => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  clearCompleted,
+}) => {
+  const todosLeft = todos.filter(todoLeft => !todoLeft.completed).length;
+  const isVisibleButton = todos.filter(todo => todo.completed).length;
```

Comment:
Consider renaming the variable `isVisibleButton` to something more descriptive, like `completedTodosCount`, since it represents the count of completed todos.